src/macaron/slsa_analyzer/provenance/loader.py

```diff
@@ -83,6 +103,13 @@ def _load_provenance_file_content(
         # Some provenances, such as Witness may not include the DSSE envelope `dsseEnvelope`
         # property but contain its value directly.
         provenance_payload = provenance.get("payload", None)
+    if not provenance_payload:
+        # GitHub Attestation.
+        # TODO Check if old method (above) actually works.
```

Comment:
I wonder why is there a change in the way we extract Github attestation?

Reply:
Perhaps @behnazh-w could weigh in on this?

Reply:
This was actually not relevant for this PR, so I've removed it.